### DISCUSSION TITLE
Running with Wine on Linux/macOS?

### DISCUSSION THREAD

**Discussion by hcguersoy** [PredatH0r/ChanSort#380] [Q&A] [answered]
Dear all,

did someone made experiences getting ChanSort running on Linux or macOS with the help of Wine?

I try it on macOS with Whisky / Wine 7.7 in a Windows 10 bottle but without any success.
Maybe I need to install additional libraries. 

Any hints and suggestions?

**Comment by PredatH0r** [comment, marked as answer]
https://github.com/PredatH0r/ChanSort

Under "System requirements / Linux" there are instructions on how to run ChanSort under Linux with Wine and Winetricks.

**Reply by hcguersoy** [reply]
Thank you very much, I simply overlooked this part of the README.
